install.ps1

```diff
@@ -635,22 +635,24 @@ function Install-AirgapTarball() {
         return
     }
 
-    if ($ExpectedImageAirGapChecksum) {
+    if (-Not $ExpectedImageAirGapChecksum) {
         return
     }
 
     $archInfo = Get-ArchitectureInfo
     $suffix = $archInfo.Suffix    
     $arch = $archInfo.Arch
 
+    if (Test-Path $InstallAgentImageDir) {
+        Remove-Item -Path $InstallAgentImageDir -Force -Recurse
```

Comment:
Why are we nuking the agent image dir? What if the user had placed some of their own image tarballs there prior to install?

Reply:
Yeah, this is new behavior as compared to the Linux install script.